### ISSUE TITLE
Test Kitchen on windows fails to upload data bags

### ISSUE THREAD

**Issue by ldooks** [test-kitchen/test-kitchen#1006]
## Description

Even after the winrm-fs 0.4.1 gem, data bag / data bag key uploads fail. If I comment out everything related to databags in my kitchen.yml, kitchen will converge successfully

## Kitchen Version

Tell us which version of test-kitchen you are using (`kitchen --version`).  

```ruby
Test Kitchen version 1.6.0
```

## ChefDK Version

If you are running test-kitchen via ChefDK, `chef --version` will provide additional relevent version details.

```ruby
Chef Development Kit Version: 0.12.0
chef-client version: 12.8.1
berks version: 4.3.0
kitchen version: 1.6.0
```

## Ruby Version

If you are not using test-kitchen via ChefDK, please provide the output of ruby --version.

```ruby
ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin15]
```

## Platform Version

Tell us which Operating System distribution and version test-kitchen is running on.

Mac OSX 10.11.4

## Replication Case

Tell us what steps to take to replicate your problem.  See [How to create a Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve)
for information on how to create a good replication case.
---
driver:
  name: vagrant

provisioner:
  name: chef_solo
  data_bags_path: databags
  encrypted_data_bag_secret_key_path: databags/encrypted_data_bag_secret

platforms:
  - name: windows-2012r2

Upon running kitchen converge, the virtual box VM will launch, install chef, copy over the cookbooks, and then die when trying to copy over data bags

## Kitchen Output

The relevant output of the test-kitchen run or a link to a gist of the entire run, if there is one.

The debug output (kitchen -l debug) may be useful, but please link to a gist, or truncate it.

Please do not paste the full run out put here.  If the full output would be useful, link to it in a gist.
https://gist.github.com/ldooks/dc821403cecd804e48092ef3b8351faa

## Kitchen Diagnose

Please include the output of `kitchen diagnose --all` or `kitchen diagnose --loader` (if the first failed).  This can be helpful in troubleshooting, so please include it in a gist.
## NOTE:

This repository is for core issues with the Test-Kitchen framework, so if the issue is with a plugin you may be referred to file the issue with the appropriate plugin.
https://gist.github.com/ldooks/183498aa8cd3a21f95b3096d0deff989

**Comment by mwrock** [comment]
Yes this is a known issue. It is fixed in Wwinrm-fs 0.4.1/Test-Kitchen 1.7.2.

**Comment by ldooks** [comment]
Thank you! Looks like my test kitchen version was out of date

confirmed i can converge
